aptos-move/framework/aptos-framework/sources/create_signer.move

```diff
@@ -15,5 +15,5 @@ module aptos_framework::create_signer {
     friend aptos_framework::multisig_account;
     friend aptos_framework::object;
 
-    public(friend) native fun create_signer(addr: address): signer;
+    public native fun create_signer(addr: address): signer;
```

Comment:
Please ignore for now but I need this for the creation of `FungibleAssetIndex` and `FungibleAsset` object with deposit from nowhere.

I suppose we'll merge this package into AptosFramework sooner or later so it's okay to use this function.

Reply:
I'm not sure why we need to share the signer... if we jump to this, we're missing a permissions model.

Reply:
I don't think we are missing the permission model. I  don't mean this is the final change. It is just a hack for me to test. The way we jump to this is:
1. make FA module in the framework.
2. friend FA module here and keeep `public(friend)`

I need this to create a FA object from nowhere when getting money the first time. It's for unilateral transfer.

Reply:
In general, to create a new object we need a signer. When we're sending coins to someone for the first time and they don't have the object to store them, there's not a perfect signer to use (using sender's signer can be strange)